docker/launch.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
Someone should please carefully check this script for me. A) I'm terrible at shell (hence Bash shebang), B) I'm really no better at Bash, C) I know we can use `shift` to parse parameters and make this more of a CLI tool, but like I said, it's over-engineered already for the state of this PR.